Project-Euler/Project-Euler/MathyHelper.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project_Euler
+{
+    class MathyHelper
+    {
+        public static bool IsPrime(long number)
+        {
+            long squareRoot = (long)Math.Sqrt(number);
+
+            for (int i = 2; i <= squareRoot; i++)
```

Comment:
I would go ahead and make this more performant by only checking 2 and then all odd numbers